src/applications/disability-benefits/all-claims/pages/index.js

```diff
@@ -113,12 +112,11 @@ import * as vaEmployee from './vaEmployee';
 import * as vaMedicalRecords from './vaMedicalRecords';
 import * as veteranInfo from './veteranInfo';
 import * as workBehaviorChanges from './workBehaviorChanges';
-import * as addDisabilitiesRevised from './addDisabilitiesRevised';
```

Comment:
One more place I'm still seeing this is in src/applications/disability-benefits/all-claims/tests/property-names.unit.spec.js